LDM-503.tex

```diff
@@ -12,7 +12,7 @@
                     
 \setDocAuthor   {William O'Mullane, Mario Juiric, Frossie Economou}                % the author(s)
 \setDocApprove  {Victor Krabendam?}              % approval by ...
-\setDocRef      {LDM-503-\docIssue} % the reference code
+\setDocRef      {LDM-503-DRAFT} % the reference code
```

Comment:
Remove `-DRAFT`. The document is now full of DRAFT messages if the `lsstdraft` class option is enabled so this is no longer needed.